pkg/build/linux.go

```diff
@@ -179,8 +181,8 @@ func runMakeImpl(arch, compiler, linker, ccache, kernelDir string, jobs int, ext
 }
 
 func runMake(params Params, extraArgs ...string) error {
-	return runMakeImpl(params.TargetArch, params.Compiler, params.Linker, params.Ccache,
-		params.KernelDir, params.BuildCPUs, extraArgs)
+	return runMakeImpl(params.TargetArch, params.Compiler, params.Linker,
```

Comment:
Do we actually need to separate `runMake` and `runMakeImpl`? Looks like `runMakeImpl` is only used here now.